tests/suites/test_suite_pkcs1_v15.function

```diff
@@ -92,10 +95,18 @@ void pkcs1_rsaes_v15_decrypt(int mod, char *input_P, char *input_Q,
     TEST_ASSERT(mbedtls_test_read_mpi(&Q, input_Q) == 0);
     TEST_ASSERT(mbedtls_test_read_mpi(&N, input_N) == 0);
     TEST_ASSERT(mbedtls_test_read_mpi(&E, input_E) == 0);
+    TEST_ASSERT(mbedtls_rsa_deduce_private_exponent(&P, &Q, &E, &D) == 0);
+
+    TEST_ASSERT(mbedtls_rsa_import(&ctx, &N, &P, &Q, &D, &E) == 0);
+
+#if !defined(MBEDTLS_RSA_NO_CRT)
+    TEST_ASSERT(mbedtls_rsa_deduce_crt(&P, &Q, &D, &DP, &DQ, &QP) == 0);
```

Comment:
Same (and all other calls in tests, I won't comment on each one).

Pre-existing: given how many times this happens, I wonder if we should have a helper function for "populate an RSA context from a set of  test input numbers".

Reply:
Notice: in order to reduce code duplication as much as possible and re-use it in different test suite files I added helper functions to `framework`. Unfortunately this changed this PR from a standalone one to being the last one of a series of 3. I updated the description accordingly.

Reply:
How about merging `test_suite_pkcs1_v15.function` and `test_suite_pkcs1_v21.function` into `test_suite_rsa.function`? (We can still keep separate `.data` files.) I never understood why RSA got three test suites when generally each module gets one.

Reply:
That's a good plan as well as it allows this PR to remove all of its dependencies